sharktank/sharktank/layers/paged_attention.py

```diff
@@ -28,12 +28,31 @@
     QuantizerTensor,
     PlanarQuantizedTensor,
     StaticScaledQuantizer,
+    TensorScaledLayout,
 )
 from sharktank import ops, kernels
 
 __all__ = ["PagedAttention"]
 
 
+def cachify(tensor):
```

Comment:
nit: can we rename this to something like maybe_unpack or use the unbox func from ops?

Reply:
unbox is different, as it performs dequantization. I am just taking out the "qs"